x/ccv/provider/client/cli/query.go

```diff
@@ -285,10 +285,10 @@ $ %s query provider registered-consumer-reward-denoms
 	return cmd
 }
 
-func CmdAllPairsValConAddrByConsumerChainID() *cobra.Command {
+func CmdAllPairsValConsAddrByConsumer() *cobra.Command {
```

Comment:
Same comment here why not use `byConsumerID`

Reply:
see above